### PR TITLE
Add baseStyle config option

### LINKED ISSUE #63: Font: Declaration In Styling Causing Issues
It's a small thing that's causing a bit of a headache, but as of right now the `font` declaration:

`font: 300 14px "Helvetica Neue", Helvetica, Arial, sans-serif;`

Is affecting all of my component's typography since it is included in `.sg` and many of its children. Would it be better to let the user decide what font-family they want to use rather than making them overwrite the base RSG styles?

### PR DESCRIPTION
**Pull request by bendman**
This fixes #63 by adding a `baseStyle` config option to the options object, also accessible on the CLI using `-b` or `--baseStyle`.  **The option defaults to `true` to be backwards compatible.**

When `true`, this includes the base stylesheet, now as a CSS file imported via a `<link>` tag.
When `false`, this skips parsing or including the base stylesheet, making it easier to developers to add their own styles without needing to override the base.

### CODE REVIEW

File: lib/fixtures/index.html.mustache [hunk 1]
```diff
@@ -11,7 +11,10 @@
     <meta name="apple-mobile-web-app-capable" content="yes">
     <meta name="format-detection" content="telephone=no">
 
-    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+    {{#baseStyle}}
+      <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
```
Comment (theogravity) [inline lib/fixtures/index.html.mustache]: do we still need font-awesome?

File: package.json [hunk 2]
```diff
@@ -81,6 +81,7 @@
     "cssnext-loader": "1.0.1",
     "deepmerge": "anthonator/deepmerge#feature/array-options",
     "express": "4.14.0",
+    "extract-text-webpack-plugin": "^1.0.1",
```
Comment (theogravity) [inline package.json]: keep the package vers fixed

**Comment by theogravity** [comment]
not seeing any overall issues, would like the package vers fixed though

File: lib/fixtures/index.html.mustache (hunk 1 above)
Comment (bendman) [inline lib/fixtures/index.html.mustache]: I wasn't sure what font-awesome was for, so I didn't include a cleanup in this change.

**Comment by bendman** [comment]
I fixed the version issue, rebased to cleanup the commit (and fix merge conflicts), and updated the history

File: lib/fixtures/index.html.mustache (hunk 1 above)
Comment (theogravity) [inline lib/fixtures/index.html.mustache]: np, leave it for now

**Comment by theogravity** [comment]
🚢

**Comment by bendman** [comment]
shipped as `3.13.0`

**Pull request merged**